ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -524,8 +526,8 @@ private void setupLanguageToggle() {
 
             for (final Locale locale : mLanguages) {
                 mLanguageTabs.addTab(mLanguageTabs.newTab()
-                                             .setText(locale.getDisplayName(locale))
-                                             .setTag(locale));
+                        .setText(locale.getDisplayName(locale))
+                        .setTag(locale));
```

Comment:
we should figure out if Android Studio has a patch mode for adding changes. All the lines before this were just formatting changes and not related to the functionality change at all. Ideally they shouldn't be included in the PR

Reply:
it looks like IntelliJ 2018.1 adds patch commit support. AS 3.1.4 is based on 2017.3.3, so it will probably be available in a version or two.

Reply:
Android Studio 3.2 is based on IntelliJ 2018.1, so it should have git patch mode.